pages/koneet-kiertoon.tsx

```diff
@@ -0,0 +1,411 @@
+  return (
+    <div>
+      <Head>
+          <title>Koneet kiertoon | Testausserveri</title>
+          <meta name="description" content="Testausserveri on kaikille avoin yhteisö koodaamisesta, eettisestä hakkeroinnista ja yleisesti teknologiasta innostuneille nuorille." />
+      </Head>
+      <Content>
+
+        <H1>Koneet kiertoon -kampanja</H1>
+        <InfoBox>
+          <span>Tämän sivun tietoja täydennetään vielä.</span>
+          <span>Kampanjan alkamisesta tiedotetaan erikseen.</span>
+        </InfoBox>
+        <p>
+          Testausserveri ry organisoi yksityisen lahjoittajan lahjoittamien tietokoneiden ja palvelimien jakamisen yhdistyksen jäsenille ja hyväntekeväisyyteen.
+          Kampanjan tavoitteena on antaa turhiksi jääneille vanhoille, mutta edelleen moniin eri käyttötarkoituksiin sopiville tietokoneille ja palvelimille uusi elämä.
+          Jaossa on toimistokäytössä olleita pöytäkoneita ja kannettavia, sekä tuotantovalmiita palvelimia.*
+        </p>
+        <Section>
+          <h2>Lahjoitettavat laitteet</h2>
+          <p>
+            Tässä on alustava lista kaikista lahjoitettavista laitteista ja niiden eri konfiguraatioista. Ilmoitetut määrät saattavat muuttua, mikäli osa laitteista on rikkinäisiä tai vaatii varaosia.
+            Kaikki laitteet jaetaan puhdistettuina ja testattuina.
+            <br />
+            <b>Kaikkien laitteiden yhteydessä annetaan kyseiseen laitteeseen sopiva virtalähde/kaapeli.</b>
+          </p>
+          <Device>
+            <DeviceAmount>Jäljellä 15 / 15 ⤬</DeviceAmount>
+            <h3>HP Workstation Z240</h3>
+            <p>
+              Nämä HP:n Workstation koneet ovat täydellisiä kevyeen jokapäiväiseen käyttöön ja palvelimeksi hurisemaan vaikka oman huoneen nurkkaan.
+              Koneet ovat hyvässä kunnossa, mutta niiden lämpötahna on suhteellisen vanhaa, joten sen vaihtamisella koneista saa varmasti lisää tehoa.
+              Koneiden koteloissa saattaa olla kaikenkokoisia pintanaarmuja.
+              Koneen eri konfiguraatiot ovat valittavissa hakulomakkeessa. Valitsethan itsellesi sopivimman vaihtoehdon ja perustele valintaasi hakemuksessa,
+              jotta tarvittaessa voidaan valita parhain mahdollinen vaihtoehtoinen konfiguraatio.
+            </p>
+            <ResponsiveSplitFlex>
+              <SpecList>
+                <h4>Tekniset tiedot</h4>
+                <p><b>Kotelo:</b> Torni</p>
+                <p><b>Suoritin:</b> Intel Core i7-7700 (3.60 GHz - 4.20 GHz, 8 Mt, 4 ydintä)</p>
+                <p>
+                  <b>Muisti:</b>
+                  <br />
+                  &nbsp;16 Gt (2x8) DDR4-2400 / DDR4-2666
+                  <br />
+                  &nbsp;4 muistikantaa, maks. 64 Gt</p>
+                <p><b>Piirisarja:</b> ?</p>
+                <p>
+                  <b>Näytönohjain:</b>
+                  <br />
```

Comment:
This section (along with "kiintolevy", "laajennuspaikat" and "liitännät" sections) should maybe be an `<ul>`. It would be semantically more correct, and I think the `&nbsp;` looks a bit weird

Reply:
This is true, but we'll fix this later.